lib/quartz/go_process.rb

```diff
@@ -42,13 +42,13 @@ def socket
   end
 
   def block_until_server_starts
-    max_retries = 10
+    max_retries = 7 # for delay of 0.005 it's about 7,69 sec
```

Comment:
Let's make the number of retries enough to match the original max delay of 102.4 seconds.